libs/sdk-ui-dashboard/src/model/store/widgetDrills/widgetDrillSelectors.ts

```diff
@@ -296,29 +297,26 @@ const selectCrossFilteringByWidgetRef: (
                 return undefined;
             }
 
-            // pick the first available attribute for identification and mandatory predicate
-            const drillAttribute = availableDrillAttributesWithoutDates[0];
-
             // collect all possible attribute names for the title
             const title = availableDrillAttributesWithoutDates
                 .map((attribute) => attribute.attribute.attributeHeader.name)
                 .join(", ");
 
+            // construct predicates for all available attributes
+            const predicates = availableDrillAttributesWithoutDates.map((drillAttribute) =>
+                HeaderPredicates.localIdentifierMatch(
+                    drillAttribute.attribute.attributeHeader.localIdentifier,
+                ),
+            );
+
             return {
                 drillDefinition: {
                     type: "crossFiltering",
                     transition: "in-place",
-                    origin: {
-                        type: "drillFromAttribute",
-                        attribute: localIdRef(drillAttribute.attribute.attributeHeader.localIdentifier),
-                    },
+                    origin: {} as DrillOrigin, // not needed for cross filtering
```

Comment:
Agreed to keep this empty as we can drill from all of the attributes by using only one drill definition.